spec/command/users_spec.rb

```diff
@@ -52,5 +52,18 @@
         invoke
       end
     end
+    describe_command 'user rotate_api_key --user non-existing' do
+      before do
+      expect(RestClient::Request).to receive(:execute).with({
+            method: :head,
+            url: "https://core.example.com/api/resources/#{account}/user/non-existing",
+            headers: {authorization: "fakeauth"},
+            username: username,
+        }).and_raise RestClient::ResourceNotFound
+      end
+      it 'rotate_api_key with non-existing --user option' do
+        expect { invoke }.to raise_error(GLI::CustomExit, /user 'non-existing' not found/i)
```

Comment:
won't this fail? should be capitalized

Reply:
Actually it passed because the trailing 'i' indicates to ignore case.
But I will capitalize the expected message to match the actual message

Reply:
Actually it passed because the trailing 'i' indicates to ignore case.
But I capitalized the 'user' to match the actual message being prompted